<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageNamesPage</class>
 <widget class="QWidget" name="ManageNamesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>565</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Manage Names</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="hboxTopToolbar">
     <item>
      <widget class="QLabel" name="labelPrepareRecords">
       <property name="text">
        <string>Additional services:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnManageDomains">
       <property name="toolTip">
        <string>Host your domains in EmerDNS system</string>
       </property>
       <property name="text">
        <string>DNS</string>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/EmerDNS-32.png</normaloff>:/icons/EmerDNS-32.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDpo">
       <property name="toolTip">
        <string>Notarize documents using blockchain.</string>
       </property>
       <property name="text">
        <string>DocNotar</string>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/DPO-32.png</normaloff>:/icons/DPO-32.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTrustedDiploma">
       <property name="toolTip">
        <string>Trusted Diploma</string>
       </property>
       <property name="text">
        <string>TD</string>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/Trusted-diploma-16-monochrome.png</normaloff>:/icons/Trusted-diploma-16-monochrome.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSsl">
       <property name="toolTip">
        <string>Trusted Diploma</string>
       </property>
       <property name="text">
        <string>SSL</string>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/EmerSSL-32.png</normaloff>:/icons/EmerSSL-32.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnInfoCard">
       <property name="toolTip">
        <string>Trusted Diploma</string>
       </property>
       <property name="text">
        <string>InfoCard</string>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/InfoCard-32.png</normaloff>:/icons/InfoCard-32.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hSpacerToolbar">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelAvailability">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QValidatedLineEdit" name="registerName">
       <property name="toolTip">
        <string>Enter name to add, update or delete</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string>Enter name to add, update or delete</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="registerTimeUnits">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set number of days that will be added to name expire time.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="txTimeTypeSelector">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>days</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>months</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>years</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="txTypeSelector">
       <property name="toolTip">
        <string>Choose operation for specified name to perform after you click on 'Submit' button</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>NAME_NEW</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NAME_UPDATE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NAME_DELETE</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelValue">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Shows how many % of maximum length used</string>
         </property>
         <property name="text">
          <string>Value:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="importValueButton">
         <property name="toolTip">
          <string>Import value from file on disk</string>
         </property>
         <property name="text">
          <string>import...</string>
         </property>
         <property name="icon">
          <iconset resource="../bitcoin.qrc">
           <normaloff>:/icons/open</normaloff>:/icons/open</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="registerValue">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Enter value for specified name. Size limit is 20Kb.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="labelAddress">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>New address:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QValidatedLineEdit" name="registerAddress">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter address/name of new owner, or leave blank to send to yourself.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string>Enter address/name of new owner of this name-value pair</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="submitNameButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This will send name_new or name_update transaction (if you already did name_new).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>&amp;Submit</string>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/send</normaloff>:/icons/send</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Names in your wallet:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>tableView</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbMyNames">
       <property name="text">
        <string>owned by me</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbOtherNames">
       <property name="text">
        <string>owned by others</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbExpired">
       <property name="text">
        <string>expired</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QValidatedLineEdit" name="nameFilter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Enter part of name to search for</string>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="placeholderText">
          <string>Name filter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QValidatedLineEdit" name="valueFilter">
         <property name="toolTip">
          <string>Enter part of value to search for</string>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="placeholderText">
          <string>Value filter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QValidatedLineEdit" name="addressFilter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Enter MFCoin address (or prefix of it)</string>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="placeholderText">
          <string>Address filter</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_ExpiresIn">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>100</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Double-click name to copy all values</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="tabKeyNavigation">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qvalidatedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
